src/main/java/net/minecraftforge/network/NetworkHooks.java

```diff
@@ -242,4 +245,46 @@ public static MCRegisterPacketHandler.ChannelList getChannelList(Connection mgr)
     {
         return mgr.channel().attr(NetworkConstants.FML_MC_REGISTRY).get();
     }
+
+    @ApiStatus.Internal
+    @Nullable
+    public static CompoundTag addAttachmentsToTag(@Nullable CompoundTag tag, ItemStack stack)
+    {
+        // Sync all serializable caps as an nbt subtag
+        CompoundTag forgeCaps = stack.serializeAttachments();
+        if (forgeCaps != null) {
+            if (tag == null)
+                tag = new CompoundTag();
+            else {
```

Comment:
Brace styles are absolutely all over the place, with new-line-braces, same-line-braces, and no-braces.